runtime/v2/shim/shim_windows.go

```diff
@@ -286,17 +284,34 @@ func openLog(ctx context.Context, id string) (io.Writer, error) {
 	return dswl, nil
 }
 
-func (l *remoteEventsPublisher) Publish(ctx context.Context, topic string, event events.Event) error {
-	ns, _ := namespaces.Namespace(ctx)
-	encoded, err := typeurl.MarshalAny(event)
-	if err != nil {
-		return err
-	}
-	data, err := encoded.Marshal()
-	if err != nil {
-		return err
+func dialer(address string, timeout time.Duration) (net.Conn, error) {
```

Comment:
Why do we need a retry look for this? On this side the containerd daemon is serving the listener so a connect should always be expected to work instantly right?

Reply:
containerd is serving this so it should always work

Reply:
I guess I mean why cant this code just say:
```go
return winio.DialPipe(address, &timeout)
```

Like it does for unix?